src/components/collections/Grid/GridRow/component.js

```
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Grid } from 'semantic-ui-react';
+
+import { DEFAULT_MAX_WIDTH } from '../constants';
+import { getTwelveColumnsWidthValue } from '../utils';
+/**
+ * GridRow is the Lodgify UI 12-columns based component for the
+ * Semantic UI Grid.Row.
+ *
+ * The [Semantic UI props](https://react.semantic-ui.com/collections/grid) can be for this component.
```

I think we should default to 'masking' the Semantic UI implementations, rather than referring to them. Is there a subset of the `Grid.Row` props which we want to expose in `GridRow`? We should be specific here.

---

Yes. Agreed.